handler/actionCurrentNews.js

```diff
@@ -15,53 +14,28 @@ export default async (chat, payload) => {
     });
 
     const push = data.results[0];
-    const report = push.reports;
 
-    if (payload.intro !== false) {
-        await chat.sendText(push.intro);
-    }
-
-    if (report.length === 1) {
-        const data = {
-            type: 'report',
-            report: report.id,
-        };
-        await chat.sendText(report[0].headline);
-        return fragmentSender(
-            chat,
-            report[0].next_fragments,
-            data,
-            report[0].text,
-            report[0].media
-        );
-    }
-
-    return sendList(chat, push);
-};
-
-const sendList = function(chat, push) {
-    const report = push.reports;
-    const elements = report.map((r) =>
-        listElement(r.headline, null,
-            buttonPostback(
-                'Lesen 📰',
-                {
-                    action: 'report_start',
-                    push: push.id,
-                    report: r.id,
-                    type: 'push',
-                }),
-            /\.(jpg|png|gif|jpeg)$/.test(r.media) ? r.media : null,
-        )
-    );
-
-    return chat.sendList(
-        elements.slice(0, 4),
-        buttonPostback(
-            'Reicht jetzt',
+    const introHeadlines = push.intro.concat('\n')
+        .concat(push.reports.map((r) => '➡ '.concat(r.headline)).join('\n'));
+    const firstReport = push.reports[0];
+    const button = buttonPostback(
+        'Alle Infos',
+        {
+            action: 'report_start',
+            push: push.id,
+            report: firstReport.id,
+            type: 'push',
+        });
+    const payloads = push.reports.map((r) =>
```

Comment:
Perhaps we should name this variable `quickReplies` or something similar, `payloads` is confusing IMO.